base_tier_validation/models/tier_validation.py

```diff
@@ -444,7 +449,7 @@ def _notify_review_requested(self, tier_reviews):
         if hasattr(self, post) and hasattr(self, subscribe):
             for rec in self.sudo():
                 users_to_notify = tier_reviews.filtered(
-                    lambda r: r.definition_id.notify_on_create and r.res_id == rec.id
+                    lambda r: r.definition_id.notify_on_create and r.res_id == rec.id  # noqa: B023
```

Comment:
try this instead:
`                    lambda r, x=rec: r.definition_id.notify_on_create and r.res_id == x.id`

Reply:
thanks